Formula/smali.rb

```diff
@@ -27,15 +27,11 @@ def install
     end
     libexec.install resource("smali"), resource("baksmali"), "smali-#{version}.jar" => "smali.jar"
 
-    inreplace "#{libexec}/smali" do |s|
-      s.gsub! /^libdir=.*$/, "libdir=\"#{libexec}\""
-    end
-    inreplace "#{libexec}/baksmali" do |s|
-      s.gsub! /^libdir=.*$/, "libdir=\"#{libexec}\""
-    end
+    inreplace libexec/"smali", /^libdir=.*$/, "libdir=\"#{libexec}\""
+    inreplace libexec/"baksmali", /^libdir=.*$/, "libdir=\"#{libexec}\""
```

Comment:
@DomT4 I suppose these should be combined since the replacements are actually the same. Another thing to add to audit! :)